src/components/rating/rating.scss

```diff
@@ -32,60 +32,68 @@
   @apply pointer-events-none;
 }
 
-.fieldset {
-  @apply m-0 flex border-0 p-0;
+fieldset {
+  margin: 0;
+  display: flex;
+  border-width: 0;
+  padding: 0;
+  margin-inline-end: var(--calcite-rating-spacing-unit);
+  align-items: center;
 }
 
 .wrapper {
-  @apply inline-block;
+  display: inline-block;
```

Comment:
Can we continue to use `@apply inline-block`? For consistency with other components. We try to use tailwind if possible for now. We may be able to get rid of tailwind at a later time.

Applies to other CSS changes in this file.

Reply:
no. I've already spoken with Franco about tailwind. This @apply is an anti-pattern in the tailwind docs itself and we will need to start moving away from it in the future. I will not be adding tech dept to this work.